### PR TITLE
Add a test and bugfix for a regression in resources

### PR DESCRIPTION
**Pull request by johnynek**
We hit this internally.

We were not testing one branch of the resources setup code.

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by johnynek** [comment]
@smparkes @dinowernli review?

File: test/BUILD [hunk 1]
```diff
@@ -140,7 +140,9 @@ java_library(
 scala_library(
     name = "ScalaLibResources",
     srcs = ["src/main/scala/scala/test/ScalaLibResources.scala"],
-    resources = ["src/main/resources/scala/test/hellos", "src/main/resources/scala/test/byes"],
+    resources = ["src/main/resources/scala/test/hellos",
+                 "src/main/resources/scala/test/byes",
```
Comment (smparkes) [inline test/BUILD]: Alphabetize while we're here?

**Comment by smparkes** [comment]
LGTM

**Comment by johnynek** [comment]
@smparkes thanks. addressed your comment (thanks for keeping me honest!)

**Comment by dinowernli** [comment]
LGTM

**Pull request merged**